### DISCUSSION TITLE
Inzidenzdaten Landkreise vor 18.11.2020

### DISCUSSION THREAD

**Discussion by krz** [marlon360/rki-covid-api#405] [General]
Hi,

über `https://api.corona-zahlen.org/districts/history/frozen-incidence/600` kriegt man die frühesten Daten vom 18.11.20. Wie kann noch ältere Daten bekommen, idealerweise seit Beginn der Pandemie im Frühjahr 2020. 
Ich weiß dass es die mal direkt vom RKI gab, aber da finde ich auch nichts mehr.

Danke!

**Comment by Rubber1Duck** [comment]
Hallo krz,
Wenn du sie nur einmalig brauchst kann ich sie dir als csv Datei bereitstellen! (eigenberechnet aus den Daten des RKI)
fühest mögliches Datum ist der 21.3.2020

**Comment by krz** [comment]
Danke @Rubber1Duck , das wäre toll. Am besten Inzidenz (7 Tage) und Todesfälle (pro 100k Einwohner) wenn du die hast.

**Comment by Rubber1Duck** [comment]
hier schonmal die Incidencen vom 21.3.2020 bis 30.11.2020 ((bzw. was frühestens möglich war zu berechnen)
[Incidence.zip](https://github.com/marlon360/rki-covid-api/files/8124238/Incidence.zip)
ohne Gewähr auf Richtigkeit.
Die Todesfälle auch auf die letzten 7 Tage bezogen (also eine Todesfall Incidence) oder einfach  kumuliert umgerechnet auf 100k Einwohner? erklär mal möglichst genau.
zu beachten ist auch das das RKI die Einwohnerzahlen bereits zwei bzw. drei mal angepasst hat!

**Comment by krz** [comment]
danke, diese Datei sieht sehr gut aus (vom 21.-23.3. sind allerdings nur 0 und NA).
Am liebsten wäre mir sowohl absolute Todesfälle pro Tag als auch umgerechnet auf 100k Einwohner, dann kann ich das mit den API Daten kombinieren. Einen rollierenden 7-Tage Durchschnitt ö.ä. kann ich zur Not selbst berechnen.

Allerdings gibt es die Todesdaten über die API erst ab 25.1.22
`https://api.corona-zahlen.org/districts/history/deaths/300`.

Habe noch das ARD Datenarchiv entdeckt, da sind aber auch nur absolute Fälle drin: https://github.com/ard-data/2020-rki-archive

**Reply by Rubber1Duck** [reply]
> 
> Allerdings gibt es die Todesdaten über die API erst ab 25.1.22 `https://api.corona-zahlen.org/districts/history/deaths/300`.
> 
das sind dann aber die dynamischen Todesfälle! das ändert sich jeden TAG! da macht eine statische Datei keinen Sinn!

Vieleicht nochmal zur Erklärung:

Das RKI meldet jeden Tag die Fälle, Todesfälle und genesenen für den Vortag.
speichert man diese Werte sind sie fixiert! Das sind die sogenannten frozen oder fixierten Zahlen!
Aber diese Fälle, Todesfälle, Genesenen sind nicht alle am Vortag erkrankt, gestorben oder genesen, sondern verteilen sich auf die Tage in der Vergangenheit, d.H. die Tagesdaten können sich jeden Tag ändern.
Wertet man das neu aus hat man die dynamischen Fälle, Todesfälle, Genesenen .....

**Comment by Rubber1Duck** [comment]
wegen der Beschränkung des RKI auf der DB ist die Abfrage über alle districts auf 30 Tage beschränkt!
wenn du die districts einzeln abfragst entfällt diese Beschränkung !

ich mache dir ein csv fertig, dauert aber ein bischen muss noch Einkaufen :-)

in wie fern kennst du dich mit sql aus? hier [https://github.com/fnogatz/RKI-Data-Diff](https://github.com/fnogatz/RKI-Data-Diff) gibt es im branch dump eine komplette Datenbank mit allen historyinformationen da kannst du alles was du möchtest rausziehen!

**Comment by krz** [comment]
danke, ist nicht super dringend. 
SQL kann ich leider nicht so gut, hab jetzt auch keine einfache Anleitung auf diesem Repo gesehen.

**Reply by Rubber1Duck** [reply]
falls du es mal ausprobieren möchtest sag bescheid ich kann dir die SQL Befehle gerne zukommen lassen! jedoch benötigt es jeden Tag eine neue bzw. aktualisierte Datenbank wie in dem Repository beschrieben ist

**Comment by Rubber1Duck** [comment]
Ich hab das mal zu einer Diskussion gewandelt, ist ja nicht wirklich ein Issue!

Noch ein Hinweis, die Incidence Daten die ich dir geschickt habe sind die sogenannten fixierten Werte, also die wie sie am Tag berichtet wurden! Diese unterscheiden sich von den dynamischen Incidencien teils deutlich! (also die wo alle späteren Änderungen einfliessen!
Ich hoffe das ist dir bewusst! Nicht das du da dynamische und fixierte Werte durcheinander würfelst!

**Comment by Rubber1Duck** [comment]
Schau mal ob du damit was anfangen kannst
[Fixed_Todesfaelle.zip](https://github.com/marlon360/rki-covid-api/files/8126523/Fixed_Todesfaelle.zip)

**Comment by Rubber1Duck** [comment]
zum Vergleich hier die dynamischen Todesfälle, das ist aber nur heute (23.02.2022) gültig!
[dyn_Todesfaelle.csv](https://github.com/marlon360/rki-covid-api/files/8127181/dyn_Todesfaelle.csv)

**Comment by Rubber1Duck** [comment]
Wenn ich noch was helfen kann sag bescheid ....

**Reply by krz** [reply]
danke, die Datei sieht sehr gut aus. Ich arbeite erstmal damit.
